object/host_system.go

```diff
@@ -32,12 +32,31 @@ type HostSystem struct {
 	InventoryPath string
 }
 
+func (h HostSystem) String() string {
```

Comment:
OK, that makes sense. But I still think it's useful to be able to print an object and get something human-readable. How about caching the result of calling `Name()` and returning that as a result of `String()`, with the default being something like `<unknown>`?